code/modules/admin/verbs/SDQL2/SDQL_2.dm

```diff
@@ -147,6 +144,10 @@
 
 
 
+/proc/SDQL_callproc(thing, procname, args_list)
```

Comment:
shunting to a proc and calling it from here is 5 times faster than using a spawn()